test/models/user.js

```diff
@@ -6,7 +6,5 @@ module.exports = (sequelize, DataTypes) => {
 		// associations can be defined here
 	};
 
-	User.Revision = User.hasPaperTrail();
```

Comment:
I had to remove this line for a couple reasons.  First, this line was causing the hooks to be registered twice for this model since this method is also called in the tests.  Second, if we call "hasPaperTrail" here, the hooks will not be compatible with the CLS.